test/arrays/slices/negativeStrideWarnings.oldR.warn+.good

```diff
@@ -1,3 +1,4 @@
+negativeStrideWarnings.chpl:13: warning: when slicing with a range with a negative stride, the sign of the stride of the original range or domain/array dimension is currently preserved, but will be negated in a future release; compile with -snewSliceRule to switch to this new rule and turn off this warning
```

Comment:
Extra warning for the same line -- is this okay?

Reply:
Our warnings sometimes do that. I consider it a lower priority to fix this.